model/oauth/bitwarden.go

```diff
@@ -79,12 +79,12 @@ func (c *Client) CreateBitwardenJWT(i *instance.Instance) (string, error) {
 				NotBefore: now - 60,
 				IssuedAt:  now,
 				ExpiresAt: now + int64(consts.AccessTokenValidityDuration.Seconds()),
-				Subject:   c.ClientID,
+				Subject:   c.CouchID,
```

Comment:
Well, I'm not really proud of `CouchID` / `ClientID` on the OAuth clients model, but we needed to call the id field differently in the JSON sent to CouchDB, and in the JSON sent to the clients, and I didn't find a better way to do that.

Here, we need to use `c.CouchID` because `c.ClientID` is empty just after we have saved the client in CouchDB.